app/presenters/start_page_content_item.rb

```diff
@@ -0,0 +1,61 @@
+class StartPageContentItem
+  attr_reader :flow_presenter
+
+  def initialize(flow_presenter)
+    @flow_presenter = flow_presenter
+  end
+
+  def payload
+    {
+      base_path: base_path,
+      title: flow_presenter.title,
+      description: flow_presenter.description,
+      details: {
+          external_related_links: external_related_links,
+          introductory_paragraph: [
+            {
+              content: flow_presenter.start_page_body,
+              content_type: 'text/govspeak'
+            }
+          ],
+          more_information: [
+            content: flow_presenter.start_page_post_body,
+            content_type: 'text/govspeak'
+          ],
+          transaction_start_link: base_path + '/y'
```

Comment:
Could we add the start_button_text field as part of this PR? My understanding is now that not being able to transfer the customised button text for child-maintenance would stop us from deploying, so it would make sense to include in this PR?